pkg/apis/dmi/v1beta1/api.proto

```diff
@@ -111,8 +111,10 @@ message MapperRegisterResponse {
 message DeviceModel {
     // Name of a device model.
     string name = 1;
+    // NameSpace of the device model.
+    string nameSpace = 2;
```

Comment:
1. The namespace is a word, you can replace it all.
2. Do we need to consider downward compatibility? Is it more appropriate to append a field rather than insert it?